src/library/DIPS.Mobile.UI/Effects/PopoverEffect/iOS/PopoverPlatformEffect.cs

```diff
@@ -0,0 +1,58 @@
+using CoreGraphics;
+using DIPS.Mobile.UI.Components.ContextMenus;
+using DIPS.Mobile.UI.Helpers.ContextMenus.iOS;
+using UIKit;
+
+namespace DIPS.Mobile.UI.Effects.PopoverEffect;
+
+public partial class PopoverPlatformEffect
```

Comment:
Jeg tenker at dette ikke er en egen effect, og heller ikke en Popover effect. En popover er et vindu som kan ha hva som helst i seg som dukker opp over skjermen. Dette er en effekt som lager en context menu når man bruker "long press". Slik jeg ser det så er dette bare en property på ContextMenuEffect som kan hete `OpenWhen`, som har to properties:
- `Pressed` 
- `LongPressed`

`Pressed` åpner når man trykker på, `LongPressed` åpner når man holder inne.

Usikre på navnet `OpenWhen`, men det er et forslag.